.spelling

```diff
@@ -224,6 +224,7 @@ PropType
 PropTypes
 ref
 refs
+createRef
```

Comment:
This isn't actually needed as the spellchecker ignores code 👍

Reply:
It kept flagging it up cause I use createRef in `UNRELEASED.md` :)

Reply:
Sorry - I wasn't clear enough. It ignores code within markdown files. So it won't flag it because you wrote ``createRef`` in `UNRELEASED.md`

Reply:
hmm, weird - the pre-commit hook was complaining about spelling, specifically talking about the use of createRef in the markdown file, but I can't replicate it now 🤷‍♂ - removed!